src/ASM/LR/ASMu2D.C

```diff
@@ -1639,7 +1639,7 @@ bool ASMu2D::getGridParameters (RealArray& prm, int dir, int nSegPerSpan) const
   // this is preferable to figuring out all element topology information
 
   std::vector<LR::Element*>::iterator el;
-  for(el=lrspline->elementBegin(); el<lrspline->elementEnd(); el++) {
+  for(el=lrspline->elementBegin(); el<lrspline->elementEnd(); ++el) {
```

Comment:
Eller bruk range-based loops først som sist: `for (LR::Element* el : lrspline->getAllElements())`

Reply:
OK, men da trengs vel ikke denne commiten først siden endringen forsvinner igjen ved neste korsvei

Reply:
neida, jeg kommer til å squashe til slutt.